hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegionServerWrapperImpl.java

```diff
@@ -398,6 +403,17 @@ public long getWALFileSize() {
     return walFileSize;
   }
 
+  @Override
+  public List<String> getWALExcludeDNs() {
+    if (excludeDatanodeManager == null) {
+      return new ArrayList<>();
```

Comment:
Why must return ArrayList here?

Reply:
Thanks, @Apache9. If NULL is returned here, there will be NPE in the ServerMetricsTmpl.jamon, which calls `for String exclude: mWrap.getWALExcludeDNs()`.

Reply:
I mean we can not use Collections.emptyList()?

Reply:
Oh, thanks, I'll change it.